packages/framer-motion/src/value/index.ts

```diff
@@ -237,7 +244,7 @@ export class MotionValue<V = any> {
         this.timeDelta = delta
     }
 
-    updateAndNotify = (v: V, render = true) => {
+    updateAndNotify = (v: V) => {
```

Comment:
This is also a breaking change, I guess, or is it non-public API?

Reply:
It isn't public